content/news/2023-02-22-news-2023-02-22-new-r-ladies-website/index.en.md

```diff
@@ -0,0 +1,103 @@
+---
+title: New R-Ladies Website
+author: Athanasia M. Mowinckel
+date: '2023-02-22'
+slug: news/2023-02-22-new-r-ladies-website
+categories:
+  - R-Ladies
+tags: []
+---
+
+
+We finally managed to release a new R-Lades website!
+
+It's been a long journey, with many people involved, and we are happy to be able to finally share with you this new site and some new content.
+
+<!--more-->
+
+There were some key issues we wanted to resolve with our webpage, that we could not easily maintain with the Wordpress site we had going:
+
+- Multilingual website 
+  - This is not the easiest to maintain with Wordpress
```

Comment:
instead of subitems why not add them in the original item?

```markdown
- Multilingual website: This is not the easiest to maintain with Wordpress.
```